plugins/portals-list.js

```diff
@@ -466,5 +468,29 @@ var setup =  function() {
     .prop("type", "text/css")
     .html('@include_string:portals-list.css@')
     .appendTo("head");
+};
 
-}
+// given counts of resonators, links and fields, calculate the available AP
+// doesn't take account AP for resonator upgrades or AP for adding mods
+window.plugin.portalslist.portalApGainMaths = function (resCount, linkCount, fieldCount) {
+  var deployAp = (8 - resCount) * DEPLOY_RESONATOR;
```

Comment:
```suggestion
  var deployAp = (8 - resCount) * window.DEPLOY_RESONATOR;
```

Use constants with the window prefix to make the dog happy instead of defining them on top